docs/visualization.rst

```diff
@@ -154,6 +161,34 @@ It also enables the coordinates along the plot axes to be updated between frames
   >>> plt.cla()
   >>> plt.close()
 
+Adding Colorbars
+----------------
+Working with the output of `ndcube.NDCube.plot` is the context of matplotlib figures and axes can be a great way of creating more complex plots.
+Here we will show two examples of home to add a colorbar.
+
+The first is simple and depends on `matplotlib.pyplot`.
+
+.. code-block:: python
+
+  >>> my_cube.plot()  # doctest: +SKIP
+  >>> plt.colorbar()  # doctest: +SKIP
+
+The second example shows how to more intricately play with `~ndcube.NDCube` visualizations and matplotlib figures and axes.
+This includes adding the output of `ndcube.NDCube.plot` to an existing axes object.
+
+.. plot::
+  :context:
+  :include-source:
+
+  >>> fig = plt.figure()  # Create a figure
+  >>> # Create WCSAxes object and then add the NDCube plot by setting the axes kwarg.
+  >>> ax = fig.add_axes([0.1, 0.1, 0.6, 0.6], projection=my_cube[0].wcs)
+  >>> ax = my_cube[0].plot(axes=ax)
+  >>> # Create the colorbar axes object and scale it by the image.
```

Comment:
This works but the fact you have to go to these lengths is a bug. We should be setting current axes and current image like map does so it's easier to do.

Reply:
To be honest, because I don't use a lot of AIA images in my own work, I'm not sure how this is usually done with `Map`.  It was the first way I found that worked.  If you think there's a simpler way based on the `Map` workflow we can put that here instead as long as it works.

Reply:
Our plot method (for 2D) should include this: https://github.com/sunpy/sunpy/blob/master/sunpy/map/mapbase.py#L2181-L2184

Reply:
It does.  For 2-D static images at least: https://github.com/sunpy/ndcube/blob/master/ndcube/mixins/plotting.py#L178

Reply:
oh well then a simple call to `plt.colorbar()` should be all you need?

Reply:
You mean as shown above?